components/prize_pool/commons/prize_pool_placement.lua

```diff
@@ -271,6 +269,22 @@ function Placement:_parseOpponents(args)
 	end)
 end
 
+function Placement:_shouldFillPlacement(opponentIndex, place)
+	-- we want at least 1 opponent present for any placement
+	if opponentIndex == 1 then
+		return true
+	-- also fill up if the fillPlaceRange option is enabled and we have an entered placeRange
+	-- and we do not yet have reached the number of Opponents for that range
+	elseif
+		self.parent.options.fillPlaceRange and place
+		and self.placeStart + opponentIndex <= self.placeEnd + 1 then
```

Comment:
```suggestion
		and self.placeStart + opponentIndex < self.placeEnd then
```
right?

Reply:
no
`a <= b + 1` is not equivalent to `a < b`

lets assume `a = b + 1` then `a <= b + 1` is correct
on the other hand `a < b` is (under the assumption) the same as `b + 1 < b` which is equivalent to `1 < 0` which is usally wrong

and the size of a slot is always `end - start + 1`
e.g. slot `3-4` would be `end - start + 1 = 4 - 3 + 1 = 2` which is correct

if wanted we could do:
```suggestion
		and opponentIndex <= slotSize then
```

and before the if define
```lua
local slotSize = self.placeEnd - self.placeStart + 1
```